src/components/Tooltip.tsx

```diff
@@ -20,20 +22,22 @@ const Tooltip: React.FC<TooltipProps> = ({
   minWidth = 300,
   maxWidth = 550,
   style,
+  popupStyle = style,
+  iconStyle,
```

Comment:
I realised that the current `style` prop only applied to the pop itself, not the icon. Now we can discriminate between them.